conmon/conmon.c

```diff
@@ -149,22 +151,30 @@ int main(int argc, char *argv[])
 
 	}
 
+	if (getcwd(cwd, sizeof(cwd)) == NULL) {
```

Comment:
can we have this and pid_file computed in Golang and pass these options down to conmon with flags?